core/services/helper/main.py

```diff
@@ -77,11 +86,32 @@ def scan_ports() -> List[ServiceInfo]:
         return list(valid_services)
 
 
-if __name__ == "__main__":
+FASTAPI = FastAPI(
+    title="Helper API",
+    description="Everybody's helper to find web services that are running in companion.",
+    default_response_class=PrettyJSONResponse,
+)
 
-    @bottle.route("/")
-    def home() -> str:
-        services = Helper.scan_ports()
-        return r"<br>".join([service.as_html() for service in services])
 
-    bottle.run(host="0.0.0.0", port=PORT)
+@FASTAPI.get(
+    "/web_services",
+    response_model=List[ServiceInfo],
+    summary="Retrieve web services found.",
+)
+@version(1, 0)
+def web_services() -> Any:
```

Comment:
Missing response_model here.

Reply:
Check here: https://github.com/bluerobotics/companion-docker/pull/215/files/48e5b6bfd35fc696521ded5c848e1e38d140bcea#diff-6dc41ee6b98e636040e7ed21f5849b814cf34e7e52b547111b81ce7db196984eR98

Reply:
Broken link

Reply:
check lines above this one.

Reply:
Cool

Reply:
Cool. Didn't see